### PR TITLE
Fitting functions for XSVS

### PR DESCRIPTION
**Pull request by sameera2004**

### CODE REVIEW

**Comment by ericdill** [comment]
closed and reopened to force travis to use the new master branch that has lmfit pinned to 0.8.3 on travis

File: skxray/core/fitting/lineshapes.py [hunk 1]
```diff
@@ -385,3 +391,100 @@ def compton(x, compton_amplitude, coherent_sct_energy,
     counts += value
 
     return counts
+
+
+def gamma_dist(bin_values, K, M):
```
Comment (ericdill) [inline skxray/core/fitting/lineshapes.py]: is it important that `K` and `M` are capitalized?

Comment (tacaswell) [inline skxray/core/fitting/lineshapes.py]: wikipedia says κ and θ  If we drop 2.7 we could use those as the variables!

https://en.wikipedia.org/wiki/Gamma_distribution

**Comment by tacaswell** [comment]
:+1: modulo the concerns about the names.

File: skxray/core/fitting/lineshapes.py (hunk 1 above)
Comment (sameera2004) [inline skxray/core/fitting/lineshapes.py]: It is not important to keep K and M capitalized, sure we can change to  κ and θ when we drop 2.7 .

Comment (tacaswell) [inline skxray/core/fitting/lineshapes.py]: I was mostly being obnoxious with by unicode comment.

**Comment by ericdill** [comment]
lgtm

**Pull request merged**